File: api/core/src/main/kotlin/data/AdminData.kt
package data

import domain.Naslov
import domain.Oseba
import kotlinx.serialization.Serializable

@Serializable
data class AdminData(
    val oseba: Oseba,
    val naslov_refs: List<NaslovData> = listOf(),
    val kontakt_refs: List<KontaktData> = listOf()
)

File: app/src/app/services/api/openapi/models/admin-data.ts
/* tslint:disable */
/* eslint-disable */
import { KontaktData } from './kontakt-data';
import { NaslovData } from './naslov-data';
import { Oseba } from './oseba';
export interface AdminData {
  kontakt_refs?: Array<KontaktData>;
  naslov_refs?: Array<NaslovData>;
  oseba?: Oseba;
}
